internal/cmd/login.go

```diff
@@ -289,9 +232,24 @@ func loginToInfra(cli *CLI, lc loginClient, loginReq *api.LoginRequest, noAgent
 	}
 
 	fmt.Fprintf(cli.Stderr, "  Logged in as %s\n", termenv.String(loginRes.Name).Bold().String())
+
+	if options.InjectUserSSHConfig {
```

Comment:
is this bleed from @dnephin 's changes?

Reply:
No, `Login` and `loginToInfra` have been merged. Since device flow isn't a login method I had to restructure the logic around getting the login access key